app/ante/ante.go

```diff
@@ -48,6 +48,9 @@ func NewAnteHandler(options HandlerOptions) sdk.AnteHandler {
 				case "/ethermint.types.v1.ExtensionOptionsWeb3Tx":
 					// handle as normal Cosmos SDK tx, except signature is checked for EIP712 representation
 					anteHandler = newLegacyCosmosAnteHandlerEip712(options)
+				case "/ethermint.types.v1.ExtensionOptionDynamicFeeTx":
```

Comment:
What's the purpose of this `extensionType`, seeing as all Cosmos tx's use dynamic fees?

Reply:
you can define the tip, but I don't think it's beneficial tbh. I'll change update the PR

Reply:
will do this as part of the migration